plutus-pab/app/Main.hs

```diff
@@ -374,7 +383,12 @@ reportContractHistoryParser =
         (ReportContractHistory <$> contractIdParser)
         (fullDesc <> progDesc "Show the state history of a smart contract.")
 
-{- Note [Use of iohk-monitoring in PAB]
```

Comment:
We adopted the "Note [xxx]" and "see note [xxx]" pattern from GHC and use it all over the place in the plutus repo, I think we should stick with it

Reply:
It's even documented! https://github.com/input-output-hk/plutus/blob/master/CONTRIBUTING.adoc#notes

Reply:
@michaelpj yeah - i'm not gonna get out of this one easily. I think i just went "oh i can use haddock anchors" and forgot about the whole notes thing - my bad :sweat_smile: